recipes/peg/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+from conans import ConanFile, tools, MSBuild
+import os
+
+class PegConan(ConanFile):
+    name = "peg"
+    license = "MIT"
+    description = "peg and leg are tools for generating recursive-descent parsers"
+    topics = ("conan", "peg", "leg", "parser")
+    homepage = "https://www.piumarta.com/software/peg/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
I can not remember what the rules are for tools... https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-installer-packages-remove-some-settings-from-their-package-id

Please follow the docs for modifying the `package_id`